packer/jobs/basic_export.py

```diff
@@ -13,32 +13,31 @@
 from ..tasks import BaseDataTask, app
 
 logger = logging.getLogger(__name__)
-
+SEP = '\t'
 
 @app.task(bind=True, base=BaseDataTask)
-def export_maxima(self: BaseDataTask, constraint):
+def basic_export(self: BaseDataTask, constraint, **custom_name):
     """
     Example task that does basic additions of two integers.
 
     :param self: Required for bind to BaseDataTask
     :param constraint: should be in job_parameters.
     """
     handle = f'{transmart_config.get("host")}/v2/observations'
-    self.update_status(Status.FETCHING, f'Getting data from observations from {handle!r}')
-    r = requests.get(url=handle,
-                     params=dict(
-                         constraint=constraint,
-                         type='clinical'),
-                     headers={
-                         'Authorization': self.request.get('Authorization')
-                     })
+    self.update_status(Status.FETCHING, f'Getting data from observations from {handle!r} for a job named {custom_name}.')
+    r = requests.post(url=handle,
+                      json={'type': 'clinical', 'constraint': constraint},
+                      headers={
+                             'Authorization': self.request.get('Authorization')
+                      })
 
     self.update_status(Status.RUNNING, 'Observations gotten, transforming.')
     obs = ObservationSet(r.json()).dataframe
 
     self.update_status(Status.RUNNING, 'Writing export to disk.')
 
     with FSHandler(self.task_id).writer as writer:
```

Comment:
You migt want to specify the file name with:
```
Content-Disposition: attachment; filename="< custom_name >"
```

Reply:
good idea! I will create a separate ticket to add it. Currently Glowing Bear has it's own way of naming files, but this would be a good improvement. 
btw, currently basic_export does not zip files, should I change it?